Makefile

```
@@ -285,6 +286,44 @@ yamcs-dict: fprime-venv ## Generate XTCE dictionary for YAMCS (requires build-ar
 	  $(UV_RUN) fprime-to-xtce "$$DICT" -o yamcs/yamcs-data/mdb/fprime.xtce.xml
 	@echo "XTCE dictionary at yamcs/yamcs-data/mdb/fprime.xtce.xml"
 
+.PHONY: yamcs-mdb
+yamcs-mdb: yamcs-dict ## Build the YAMCS Mission Database (alias for yamcs-dict, runs after build)
+
+.PHONY: yamcs-build-check
+yamcs-build-check: ## Validate YAMCS server boots via docker compose with the current MDB
```

checkmake (0.3.2)

[warning] Target body for "yamcs-build-check" exceeds allowed length of 5 lines (27).

(maxbodylength)